tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Debugger/ProbesTests.cs

```diff
@@ -50,6 +50,7 @@ public ProbesTests(ITestOutputHelper output)
     {
         SetServiceVersion("1.0.0");
         _rcmPath = Path.Combine(EnvironmentHelper.GetSampleProjectDirectory(), RemoteConfigurationFileName);
+        SetupRcmConfiguration(_rcmPath);
```

Comment:
I'm surprised you needed to update this. If this change is needed to run the samples, then this is fine by me. Otherwise, I'd check with the Debugger folks to sign off

Reply:
I'll contact Debugger team, but basically I did this to ensure no Remote Config from other tests leak into this one.

Reply:
cc @shurivich

Reply:
I really like the idea to move implementation of `SetRcmConfiguration` to base class, however, I have some comments, I'll write inline on files.